README.md

```diff
@@ -2,9 +2,13 @@
  Basic Admin Page Using Django
 
 Useful Links:
-https://docs.djangoproject.com/en/4.1/intro/tutorial01/
+- https://docs.djangoproject.com/en/4.1/intro/tutorial01/
 
-https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-DATABASES:~:text=When%20connecting%20to%20other%20database%20backends%2C%20such%20as%20MariaDB%2C%20MySQL%2C%20Oracle%2C%20or%20PostgreSQL%2C%20additional%20connection%20parameters%20will%20be%20required.%20See%20the%20ENGINE%20setting%20below%20on%20how%20to%20specify%20other%20database%20types.%20This%20example%20is%20for%20PostgreSQL%3A
+- https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-DATABASES:~:text=When%20connecting%20to%20other%20database%20backends%2C%20such%20as%20MariaDB%2C%20MySQL%2C%20Oracle%2C%20or%20PostgreSQL%2C%20additional%20connection%20parameters%20will%20be%20required.%20See%20the%20ENGINE%20setting%20below%20on%20how%20to%20specify%20other%20database%20types.%20This%20example%20is%20for%20PostgreSQL%3A
 
 For front end:
 https://youtu.be/eosXCVcgtPw
+
+Required Installations:
+- pip install crispy-bootstrap5
```

Comment:
I recommend putting these in a `requirements.txt`, which you can generate with `pip freeze > requirements.txt` . Then document that the way we want to install the packages is with `pip install -r requirements.txt`